docs/server/v20/server/configuration/README.md

```diff
@@ -66,9 +68,13 @@ If more than one method is used to configure the server, it might be hard to fin
 
 When you run EventStoreDB with this option, it will print out the effective configuration applied from all available sources (default and custom configuration file, environment variables and command line parameters) and print it out to the console.
 
+::: warning
+If you never run EventStoreDB on Linux as a service, when starting the service executable with `--what-if` option, the server _will create the data and log directories_ owned by the current user. It will prevent the service from running properly.
```

Comment:
Isn't this doc both for Linux and Windows? If yes, then we should update the wording about `Linux as service` - I assume that this will be the same case for them?

Reply:
think so , though on windows , the owner will be the user who created the directories / files  ,
might be mitigated by permission inheritance on windows .

Reply:
Windows services normally run with administrative permissions on the server. So, the service has access to any directory on the machine. Plus, we have no Windows service support, so we hardly know how people will run it. As such, there is no such thing now as "running ESDB as a service on Windows" out of the box. Therefore, I find the issue irrelevant.